### PR TITLE
exclude build env from cov reporting

### PR DESCRIPTION
**Pull request by wimglenn**
coverage is hunting around in there (see https://github.com/python/importlib_metadata/runs/5617691032?check_suite_focus=true for example)

### CODE REVIEW

**Comment by jaraco** [comment]
Do we know why coverage is hunting around in there? Is there a reason coverage doesn't fix this for everybody?

File: .coveragerc [hunk 1]
```diff
@@ -2,6 +2,7 @@
 omit =
 	# leading `*/` for pytest-dev/pytest-cov#456
 	*/.tox/*
+        */pep517-build-env-*
```
Comment (jaraco) [inline .coveragerc:5]:
```suggestion
	*/pep517-build-env-*
```

**Comment by wimglenn** [comment]
I'm not sure if it could be addressed in coverage directly, that seems like a question for Ned

**Comment by jaraco** [comment]
I'll first try to determine whether the issue is present in coverage proper or only in pytest-cov.

**Pull request merged**